### ISSUE TITLE
"Missing Texture" in world map "Industrial Island"

### ISSUE THREAD

**Issue by iahung2** [SuperTux/supertux#2773]
**SuperTux version:** SuperTux Nightly 2024-02-05 16:50 Windows (64 bit)
**System information:** Windows 8.1 64 bit, Intel i5-4460 integrated graphics

##### Additional debugging information

Broken for both `opengl33` and `sdl` video system.

**Comment by Vankata453** [comment]
Isn't this rather an issue with the add-on itself?

**Comment by iahung2** [comment]
> Isn't this rather an issue with the add-on itself?

As I recall, not happening with previous Nightly and the stable version.

**Comment by tylerandari13** [comment]
> Isn't this rather an issue with the add-on itself?

To be fair where else are you supposed to open an issue with an addon?

**Comment by iahung2** [comment]
One screenshot is better than thousand words. How could you play with this?

![Capture](https://github.com/SuperTux/supertux/assets/156915310/e0b842bd-4bc4-487a-95f1-e35cdd7a7de7)

**Comment by Miner34dev** [comment]
> As I recall, not happening with previous Nightly

What do you mean for "previous nightly"? A couple months ago i played it and the problem was already there.
Anyways, i will give a look to why this happens when i have time.

**Comment by Vankata453** [comment]
I checked, and the cause of this issue is the tileset of the add-on having a full copy of the main tileset ("images/tiles.strf") in its state from v0.6.3, which makes it now use no-longer-existent tileset image files (since they were moved to a `deprecated/` sub-directory because of their deprecation in 517005b4ced4f005224071ae743116297d941865).

Custom tilesets, which wish to utilize one or more other tilesets, should use `import-tileset` instead to combat this issue. Copying tilesets over is a disaster for compatibility.
